### ISSUE TITLE
no file

### ISSUE THREAD

**Issue by charlesRivias** [deepjyoti30/ytmdl#262]
after download no files exist in dir

Downloaded!
 ==> Getting song data for https://music.youtube.com/watch?v=o_ryMz8OLxY...
 ==> Something went wrong with iTunes. The program will continue withthe other providers. Please check '[C:\Users\RAVIXVI\.cache\ytmdl\logs\log.cat]' for more details.
Couldn't read cache at: .cache
Couldn't write token to cache at: .cache
 ==> Moved to D:\...
 ==> Done

**Comment by pimpmobile01** [comment]
same issue

**Comment by deepjyoti30** [comment]
@pimpmobile01 Are you able to reproduce the issue on a Linux machine or is it an Windows machine? @charlesRivias seems to be using Windows.

I just confirmed and this issue is not producable on Linux. I think it's specific to Windows.

**Comment by PanConKisu** [comment]
I got the same issue and I fixed it by myself
If you are tryng to download a song directly by the youtube link, use the --url flag

example: ytmdl --url https://www.youtube.com/watch?v=7MCoMI0I5IU

(sorry if my english is bad, spanish is my main language and im too lazy to use a translator.)

**Comment by PanConKisu** [comment]
oh and use the --skip-meta to avoid the Something went wrong with iTunes. thing.

**Comment by R-Satwik** [comment]
Yes, only put --url infront of the actual url. Its searching for the metadata of url instead of song name (which returns an error)
eg: ytmdl --url URL

if you dont put --url, it will ignore all the flags after the url and returns an itunes error and will delete your file
